js/mainPage.js

```diff
@@ -0,0 +1,226 @@
+
+function loadSlider(){
+	//create the slider
+	var slider = document.getElementById('mySlider');
+	noUiSlider.create(slider, {
+    	start: [0, 100],
+    	connect: true,
+    	range: {
+        	'min': 0,
+        	'max': 100
+	    }
+	});
+	//event for the display of the slider range
+	slider.noUiSlider.on('update',function(values,handler){
+	document.getElementById("first").innerHTML="$"+values[0];
+	document.getElementById("second").innerHTML="$"+values[1];
+	$('.menu').innerHTML = "";	
+	loadElements(parseInt(values[0]),parseInt(values[1]));
+});
+	var ok=localStorage.getItem("itemsLength");
```

Comment:
what does `ok` stand for? also the indentation seems wrong

Reply:
Is that variable used?

Reply:
It is a "leftover" from something else, I thought that it was deleted 😓